plugins/repository-hdfs/src/main/plugin-metadata/plugin-security.policy

```diff
@@ -95,4 +95,7 @@ grant {
   // client binds to the address returned from the host name of any principal set up as a service principal
   // org.apache.hadoop.ipc.Client.Connection.setupConnection
   permission java.net.SocketPermission "localhost:0", "listen,resolve";
+
+  // allow to access krb5.conf
+  permission java.io.FilePermission "${{java.security.krb5.conf}}", "read";
```

Comment:
@reta should we consider making this a general grant within `security.policy`? This issue impacts the security plugin too: https://github.com/opensearch-project/security/issues/5646